taxcalc/tests/test_policy.py

```diff
@@ -30,45 +53,17 @@ def test_correct_class_instantiation():
     pol = Policy()
     assert pol
     pol.implement_reform({})
-    with pytest.raises(ValueError):
+    with pytest.raises(pt.ValidationError):
         pol.implement_reform(list())
-    with pytest.raises(ValueError):
+    with pytest.raises(pt.ValidationError):
         pol.implement_reform({2099: {'II_em': 99000}})
     pol.set_year(2019)
-    with pytest.raises(ValueError):
+    with pytest.raises(pt.ValidationError):
         pol.implement_reform({2018: {'II_em': 99000}})
-    with pytest.raises(ValueError):
+    with pytest.raises(pt.ValidationError):
         pol.implement_reform({2020: {'II_em': -1000}})
 
 
-def test_policy_json_content_consistency():
-    """
-    Test contents of JSON defaults file, which is read into Policy._vals dict.
-    """
-    expected_vi_vals = {
-        'MARS': ['single', 'mjoint', 'mseparate', 'headhh', 'widow'],
-        'EIC': ['0kids', '1kid', '2kids', '3+kids'],
-        'idedtype': ['med', 'sltx', 'retx', 'cas', 'misc', 'int', 'char']
-    }
-    policy = Policy()
-    start_year = policy.start_year
-    assert start_year == Policy.JSON_START_YEAR
-    policy_vals = getattr(policy, '_vals')
-    for name, data in policy_vals.items():
-        # check entries in value_yrs list
-        value_yrs = data['value_yrs']
-        assert isinstance(value_yrs, list)
-        value = data['value']
-        expected_value_yrs = [(start_year + i) for i in range(len(value))]
-        if value_yrs != expected_value_yrs:
-            msg = 'name,value_yrs,expected_value_yrs: {}\n{}\n{}'
-            raise ValueError(msg.format(name, value_yrs, expected_value_yrs))
-        # check entries in vi_vals list
-        vivals = data['vi_vals']
-        if vivals:
-            assert vivals == expected_vi_vals[data['vi_name']]
```

Comment:
`value_yrs` is no longer used, and label validation on the `vi_vals` is done when the defaults are loaded.